### PR TITLE
add metaspace config

### PR DESCRIPTION
**Pull request by Alexkun**
### What is the purpose of the change

Add metaspace jvm config for Ecs

### Related issues/PRs

Related issues: #4053

### Brief change log

- Define the default jvm metaspace size for Ecs.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-core/src/main/scala/org/apache/linkis/manager/engineplugin/common/conf/EnvConfiguration.scala [hunk 1]
```diff
@@ -41,13 +41,15 @@ object EnvConfiguration {
   val ENGINE_CONN_CLASSPATH_FILES =
     CommonVars("wds.linkis.engineConn.files", "", "engineConn额外的配置文件")
 
-  val metaspaceSize = if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_1_8)) {
-    "-XX:MaxMetaspaceSize=256m -XX:MetaspaceSize=128m"
+  val MAX_METASPACE_SIZE = CommonVars("linkis.engineconn.metaspace.size.max", "256m")
+
+  lazy val metaspaceSize = if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_1_8)) {
```
Comment (jackxu2011) [inline linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-core/src/main/scala/org/apache/linkis/manager/engineplugin/common/conf/EnvConfiguration.scala:46]: maybe the judge can be deleted, the min jdk is required 1.8

**Review by jackxu2011 — CHANGES_REQUESTED** [review]

File: linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-core/src/main/scala/org/apache/linkis/manager/engineplugin/common/conf/EnvConfiguration.scala (hunk 1 above)
Comment (peacewong) [reply linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-core/src/main/scala/org/apache/linkis/manager/engineplugin/common/conf/EnvConfiguration.scala:46]: yes

**Review by peacewong — APPROVED** [review]
LGTM.

**Review by jackxu2011 — APPROVED** [review]
LGTM

**Pull request merged**